components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -136,46 +149,34 @@ export default function Preferences() {
                 <Subheading>Workspaces will stop after a period of inactivity without any user input.</Subheading>
 
                 <div className="mt-4 max-w-xl">
-                    {!maySetTimeout.isLoading && maySetTimeout.data === false && (
-                        <Alert type="message">
-                            Upgrade organization{" "}
-                            <Link to="/billing" className="gp-link">
-                                billing
-                            </Link>{" "}
-                            plan to use a custom inactivity timeout.
-                        </Alert>
-                    )}
-
-                    {maySetTimeout.data === true && (
-                        <form onSubmit={saveWorkspaceTimeout}>
-                            <InputField
-                                label="Default Workspace Timeout"
-                                hint={
-                                    <span>
-                                        Use minutes or hours, like <span className="font-semibold">30m</span> or{" "}
-                                        <span className="font-semibold">2h</span>
-                                    </span>
-                                }
-                            >
-                                <div className="flex space-x-2">
-                                    <div className="flex-grow">
-                                        <TextInput
-                                            value={workspaceTimeout}
-                                            placeholder="e.g. 30m"
-                                            onChange={setWorkspaceTimeout}
-                                        />
-                                    </div>
-                                    <Button
-                                        htmlType="submit"
-                                        loading={timeoutUpdating}
-                                        disabled={workspaceTimeout === user?.additionalData?.workspaceTimeout ?? ""}
-                                    >
-                                        Save
-                                    </Button>
+                    <form onSubmit={saveWorkspaceTimeout}>
+                        <InputField
+                            label="Default Workspace Timeout"
+                            hint={
+                                <span>
+                                    Use minutes or hours, like <span className="font-semibold">30m</span> or{" "}
+                                    <span className="font-semibold">2h</span>
```

Comment:
**suggestion**: We should definitely get rid of the ambiguity here cause by the dual metric and default to minutes, which is closer to real use cases and significantly decreases cognitive load—every time I see this help text I feel like I have to solve a quadratic equation.